app/src/main/java/com/arygm/quickfix/ui/theme/Theme.kt

```diff
@@ -20,61 +20,105 @@ import androidx.compose.ui.unit.sp
 import androidx.core.view.WindowCompat
 import com.arygm.quickfix.R
 
-private val DarkColorScheme =
-    darkColorScheme(
-        background = DarkSlateGray,
-        primary = DarkBordeaux,
-        secondary = RosyBrown,
-        tertiary = Silver,
-        error = IndianRed)
-
 private val LightColorScheme =
     lightColorScheme(
-        background = White,
-        primary = Orange,
-        secondary = Bordeaux,
-        tertiary = LightGray,
-        error = Tomato,
-    )
+        primary = ButtonPrimary,
+        secondary = ButtonSecondary,
+        tertiary = ButtonTertiary,
+        background = BackgroundPrimary,
+        surface = BackgroundSecondary,
+        onPrimary = TextButtonPrimary,
+        onSecondary = TextButtonSecondary,
+        onTertiary = TextButtonTertiary,
+        error = AccentPrimary,
+        onError = AccentSecondary,
+        onBackground = TextPrimary,
+        onSurface = TextSecondary,
+        outline = TitlePrimary,
+        surfaceVariant = ButtonQuaternary, // Fourth button color
+        onSurfaceVariant = TextButtonQuaternary, // Text color for the quaternary button,
+        onSecondaryContainer = TextDisabled,
+        tertiaryContainer = ButtonDisabled)
```

Comment:
I absolutely love how you took into account every color. This is a great addition!